zb-db/src/main/java/io/camunda/zeebe/db/ZeebeDbFactory.java

```diff
@@ -15,7 +16,7 @@
  *
  * @param <ColumnFamilyNames> the names of the column families
  */
-public interface ZeebeDbFactory<ColumnFamilyNames extends Enum<ColumnFamilyNames>> {
+public interface ZeebeDbFactory<ColumnFamilyNames extends Enum<? extends EnumValue> & EnumValue> {
```

Comment:
:thinking_face: breaking change with zpt and zdb ? I guess no because we also use the same enum

Reply:
🤔 Perhaps, but it's worth a breaking change